### DISCUSSION TITLE
Recover from server crash

### DISCUSSION THREAD

**Discussion by onetimepassword** [gogs/gogs#7464] [Q&A]
I have the raw /var/lib/gogs/repos/... .git folders recovered from a server that crashed and would like to import them into a new gogs install. Any suggestion how I would accomplish that?

**Comment by annpocoyo** [comment]
The below is copy and paste from one of my other answers. I know I am a bit late but might help other people.

You can run it as a local repository. You should just be able to specify the git repository folder as a path to git clone like this:`git clone path/to/gogs-repository/user/repo.git/`. Then fire up a new gogs instance and create and push the just cloned git repository to it.
